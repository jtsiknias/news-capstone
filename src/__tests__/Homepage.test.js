import { render } from '@testing-library/react';
import Homepage from '../pages/Homepage';

it('renders the Homepage component without crashing', () => {
  render(<Homepage />);
});

it('matches snapshot', () => {
  const { asFragment } = render(<Homepage />);
  expect(asFragment()).toMatchSnapshot();
});
