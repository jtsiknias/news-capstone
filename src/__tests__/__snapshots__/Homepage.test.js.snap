// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="my-5 container-fluid"
  >
    <div
      class="row"
    >
      <div
        class="px-4 col-lg-3 col-12"
      >
        <div
          class="list-group list-group-flush"
        />
      </div>
      <div
        class="border-left border-right px-4 col-lg-6 col-12"
      />
      <div
        class="px-4 col-lg-3 col-12"
      >
        <div
          class="list-group list-group-flush"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
